### ISSUE TITLE
cracking in mp3 only not in spotify 

### ISSUE THREAD

**Issue by SchmidtBoris** [jwallet/spy-spotify#56]
**What is the bug ?**
> A clear and concise description of what the bug is.
LP like cracking or jitter in mp3 only not in spotify play (v1.1.3.0)
**How to reproduce it or what's the expected behavior ?**
> Steps to reproduce the behavior or describe the expected behavior:
record songs with silent passages
adding EspionSpotify to no defender task list does not help !
**Screenshots**
> If applicable, add screenshots to help explain your problem.
![grafik](https://user-images.githubusercontent.com/8724629/46684147-441bd900-cbf2-11e8-868d-b6ec16d00c56.png)

**Comment by SchmidtBoris** [comment]
There is no cracking in the recorded wav file. Also cracking with new libmp3lame.64.dll 3.100
Looks like there are some bad parameters by controlling the libmp3lame.64.dll

**Comment by jwallet** [comment]
if by cracking you mean, you have pop sounds recorded in your mp3 files, look at this closed issue #43, you have to set both apps with the same bitrate, no spotify premium, no 320kbps

**Comment by jwallet** [comment]
i will consider it fixed then

**Comment by hulzer** [comment]
It seems to be a power management problem.

I found a solution here:
https://www.drivereasy.com/knowledge/audio-or-sound-popping-on-windows-107-solved/

A changed the minimum processor state to 100 and now all pop sound are gone.

**Comment by jwallet** [comment]
I don't test on a laptop, so i would never found out.

**Comment by hulzer** [comment]
It happened on a desktop...

**Comment by holdit** [comment]
@jwallet Just wanted to confirm what @hulzer said. I was also having this issue on my computers (both desktops, running Windows 7 and 10) and was only able to fix it by setting the power plan to "high performance" or by changing the minimum processor state to 100.

It may be worth adding this to the FAQs.

**Comment by jwallet** [comment]
not sure if this trick fixed your issue, mine is still set to 5% and I don't have this issue 
![image](https://user-images.githubusercontent.com/23088305/53061589-12622b00-348c-11e9-9fe0-8a7f70ed3c1d.png)

**Comment by holdit** [comment]
It did. I've just finished recording ~400 songs, had to redo ~30 because the start/finish was messed up but to cracking sounds.

I don't know what the issue is to be fair. It happens on Win 7 and 10 for me so maybe it's hardware related?

Anyway, even if changing the "processor state" to 100% has nothing to do with the sound/recording, it did fix the issue for me (I had tried everything, triple checked the sounds, spytify/spotify bitrate, etc).
